fluent/fluentBuilder.go

```diff
@@ -0,0 +1,193 @@
+/*
+	The fluent package offers helper utilities for using NodeAssembler
```

Comment:
should this go in a doc.go or somewhere else? it's good information that people are likely to miss